posthog/models/event/query_event_list.py

```diff
@@ -52,21 +54,21 @@ def query_events_list(
     request_get_query_dict: Dict,
     order_by: List[str],
     action_id: Optional[str],
-    long_date_from: bool = False,
+    select: Optional[List[str]],
+    where: Optional[List[str]],
+    unbounded_date_from: bool = False,
     limit: int = 100,
-) -> List:
+) -> Union[List, dict]:
```

Comment:
The return type here is very opaque.

Could we use a `@dataclass` or give the types aliases?